service/tablet_allocator.cc

```diff
@@ -804,7 +804,9 @@ class tablet_allocator_impl : public tablet_allocator::impl
             : _config(std::move(cfg))
             , _migration_notifier(mn)
             , _db(db) {
-        (void)_config;
+        if (_config.initial_tablets_scale == 0) {
+            throw std::runtime_error("Initial tablets scale must be positive");
```

Comment:
nit: 0 is positive, the error should say "positive, non-zero".

Reply:
https://www.mathopenref.com/positive-number.html:
> Positive [numbers](https://www.mathopenref.com/number.html) are numbers that are greater than zero.

Reply:
:laughing: 

Well played, queued.